src/index.html

```diff
@@ -1,18 +1,70 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <section>
+      <article
+        class="product-card"
+        data-qa="card"
+      >
+        <h3 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="product-card__code-product">Product code: 195434</p>
+
+        <div class="product-card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="product-card__reviews-paragraph">Reviews: 5</p>
+        </div>
+
+        <div class="product-card__price">
+          <p class="product-card__price-label">Price:</p>
+
+          <p class="product-card__price-value">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="product-card__buy"
+          data-qa="hover"
+        >
+          Buy
+        </a>
```

Comment:
The `href` attribute in the anchor tag should lead to a real or meaningful destination if possible, even if it's a placeholder link. For example, it could be set to `#buy` to indicate an action to buy the product.